app/components/Footer/style.module.css

```diff
@@ -0,0 +1,73 @@
+.footer {
+  background: var(--gradiente-light);
+  word-break: break-word;
+  padding-top: 20rem;
+  margin-top: -16rem;
+  z-index: 0;
+  position: relative;
+}
+
+.infoContainer {
+  padding: 4rem 0;
+}
+
+.infoContainer,
+.infoContainer div,
+.infoContainer address {
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+}
+
+.infoContainer div,
+.infoContainer address {
+  gap: 0.5rem;
+}
+
+.infoContainer a:hover,
+.infoContainer a:active {
+  text-decoration: underline;
+}
+
+.infoContainer a:visited {
+  text-decoration: none;
+}
+
+.infoSubtitle {
+  color: var(--neutral-gray-darkest);
+  text-wrap: wrap;
+}
+
+.infoField {
+  display: inline-flex;
+  gap: 0.5rem;
+
+  font-style: normal;
```

Comment:
Eu tiraria isso daqui e colocaria o address no reset css como address - font: inherit

Reply:
Colocando apenas como address não redefine o formato itálico padrão do navegador. Pra poder redefinir precisa ser mais especifíco, então o código fica assim:
`address p,
address a {
  font-style: normal;
}`
Acha que fica melhor? Essa especificidade não fica melhor no arquivo de estilos do footer?

Reply:
Will, fiz um testes com ```address { font: inherit }``` e funcionou

Reply:
Coloquei ;)